packages/grid/src/components/column/Column.types.ts

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from 'react';
+
+export type ColumnProps = {
+  children: ReactNode;
+  widths: number[];
+  className?: string;
+  element?: keyof JSX.IntrinsicElements;
```

Comment:
do not forget to write a short description above each file, it will be generated for the docs at the storybook